easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.06-foss-2023b.eb

```
@@ -95,8 +95,12 @@ exts_list = [
     ('filehash', '2.4-5', {
         'checksums': ['3b1ee2794dd61e525ee44db16611c65957691d77bb26ae481eba988bb55da22c'],
     }),
-    ('ff', '4.0.12', {
-        'checksums': ['08af355a9a10fe29d48d085abc7cf1f975e1a4a670668a4f8d9632d087fb41bf'],
+    ('ff', '4.5.0', {
+        'patches': ['ff-4.5.0-riscv64-known-arch.patch'],
+        'checksums': [
+            {'ff_4.5.0.tar.gz': '2aed9052a93ca81af4c8cc7defce0f7aa0206b442b4795b6219f6e56b341188e'},
+            {'ff-4.5.0-riscv64-known-arch.patch': 'cdc20ec31d39f14332897dc630f460697efb4863751dcba659bb4129c4100dba'},
```

Sorry, only noticed this now, but I think it's better to not change the version here. Looking at the patch and the sources of ff 4.0.12 and 4.5.0, it should (probably) work fine for the older version too, at least the file that's being patches didn't change. So I'd propose that you change this to:
```suggestion
    ('ff', '4.0.12', {
        'patches': ['ff-4.0.12-riscv64-known-arch.patch'],
        'checksums': [
            {'ff_4.0.12.tar.gz': '08af355a9a10fe29d48d085abc7cf1f975e1a4a670668a4f8d9632d087fb41bf'},
            {'ff-4.0.12-riscv64-known-arch.patch': 'cdc20ec31d39f14332897dc630f460697efb4863751dcba659bb4129c4100dba'},
```

and also rename the patch file itself accordingly.